modules/postgresql/src/main/java/org/testcontainers/containers/PgroutingContainerProvider.java

```diff
@@ -0,0 +1,36 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.jdbc.ConnectionUrl;
+
+/**
+ * Factory for PgRouting container.
+ */
+public class PgroutingContainerProvider extends JdbcDatabaseContainerProvider {
+
+    private static final String NAME = "pgrouting";
+    private static final String DEFAULT_TAG = "11-2.5-2.6";
+    private static final String DEFAULT_IMAGE = "konturio/pgrouting";
```

Comment:
Hmm, if we merge this then I'd want to know that we're using a mainstream container image.

Why is [konturio/pgrouting](https://hub.docker.com/r/konturio/pgrouting) the image used here?

Reply:
There is this image, which seems official:
https://hub.docker.com/r/pgrouting/pgrouting